canal/cyclone.py

```diff
@@ -1041,10 +1041,17 @@ def get_imran_sb_wires(num_tracks: int) -> List[Tuple[int,
 
         for track in range(num_tracks):
             # f_e1
-            result.append((track, SwitchBoxSide.WEST,
```

Comment:
I'd prefer you create a new topology function called `ImranEven` or something like that and don't touch this function.